### DISCUSSION TITLE
Web Scraping in R Studio

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#592] [Q&A] [answered]
Bagaimana cara scraping website  https://scamalytics.com/ip/157.230.243.104  menggunakan R studio ? 
Saya ingin mengambil data latitude dan longitudenya dari website tersebut.

 Mohon bantuannya kak, terima kasih.

**Comment by amaliaasf** [comment, marked as answer]
Dear Pak @yohanespm77

Untuk scrapping website menggunakan R, package yang dapat membantu adalah `rvest`.

Berikut referensi yang bisa Bapak tinjau:
• Official Docummentation: https://rvest.tidyverse.org/
• https://cran.r-project.org/web/packages/rvest/rvest.pdf
• https://www.youtube.com/watch?v=v8Yh_40E-Fs

Demikian untuk jawaban sementara yang dapat kami sampaikan. Silahkan boleh dicoba terlebih dahulu Pak, kalau ada kendala boleh diinfokan kembali.

**Reply by yohanespm77** [reply]
Wahh terima kasih banyak kak @amaliaasf . Referensinya sangat membantu 🙏
